flink-dist/src/test/java/org/apache/flink/dist/BashJavaUtilsITCase.java

```diff
@@ -133,6 +133,46 @@ void testGetConfiguration() throws Exception {
         assertThat(lines).hasSize(expectedResultLines);
     }
 
+    @Test
+    void testMigrateLegacyConfigToStandardYaml() throws Exception {
+        int expectedResultLines = 26;
+        String[] commands = {
+            RUN_BASH_JAVA_UTILS_CMD_SCRIPT,
+            BashJavaUtils.Command.MIGRATE_LEGACY_FLINK_CONFIGURATION_TO_STANDARD_YAML.toString(),
+            String.valueOf(expectedResultLines)
+        };
+        List<String> lines = Arrays.asList(executeScript(commands).split(System.lineSeparator()));
+        assertThat(lines)
```

Comment:
Is it possible that the result be unstable and make the test unstable? e.g. keys are organized in a different order.